superset-frontend/src/views/CRUD/data/dataset/DatasetLayout/index.tsx

```diff
@@ -45,17 +46,19 @@ export default function DatasetLayout({
   datasetPanel,
   rightPanel,
   footer,
+  editPageIsVisible,
 }: DatasetLayoutProps) {
   return (
     <StyledLayoutWrapper data-test="dataset-layout-wrapper">
       {header && <StyledLayoutHeader>{header}</StyledLayoutHeader>}
       <OuterRow>
-        <LeftColumn>
-          {leftPanel && (
-            <StyledLayoutLeftPanel>{leftPanel}</StyledLayoutLeftPanel>
-          )}
-        </LeftColumn>
-
+        {!editPageIsVisible && (
+          <LeftColumn>
+            {leftPanel && (
+              <StyledLayoutLeftPanel>{leftPanel}</StyledLayoutLeftPanel>
+            )}
+          </LeftColumn>
+        )}
```

Comment:
Since `leftPanel` will be `null` if `editPageIsVisible` is true, could we avoid passing `editPageIsVisible` as a prop and instead simplify to this?
```suggestion
        {leftPanel && (
          <LeftColumn>
            <StyledLayoutLeftPanel>{leftPanel}</StyledLayoutLeftPanel>
          </LeftColumn>
        )}
```

Reply:
Good call, updated in [`this commit`](https://github.com/apache/superset/pull/22043/commits/c6a9b0766df0321ff07e78973aeef82dc32ffbef).